support/environment.go

```diff
@@ -79,6 +87,10 @@ func GetPyTorchImage() string {
 	return lookupEnvOrDefault(CodeFlareTestPyTorchImage, "pytorch/pytorch:1.11.0-cuda11.3-cudnn8-runtime")
 }
 
+func GetNotebookImage() string {
+	return lookupEnvOrDefault(NotebookImage, "quay.io/modh/odh-generic-data-science-notebook@sha256:bb33abc67af1328d3b32899f58bcdc0cf1681605e1b5da57f8fe8da81523a9bd")
```

Comment:
I suppose this notebook image will change between releases.
I think it would be good to not provide default value.

Reply:
Also, is this property usable outside of ODH DW repo? I think it is meaningful just there. Can you move it to that repo?

Reply:
@sutaakar I have updated GetNotebookImage to avoid using default image, but in this case `NOTEBOOK_IMAGE` should be mandatory environment variable to run this test ..
Thank you for reviewing!

> I suppose this notebook image will change between releases. I think it would be good to not provide default value.
@sutaakar I have updated GetNoteBookImage function accordingly, but in this case `Notebook_image` variable must be mandatory before running this test
Thank you for reviewing!

Reply:
Removed GetNotebookImage function and placed in distributed-workloads repository :+1: